packages/paste-website/src/pages/core/index.mdx

```diff
@@ -0,0 +1,214 @@
+---
+title: Core
+description: The Core package includes all components and libraries in Paste.
+slug: /core/
+---
+
+import {graphql} from 'gatsby';
+import {Separator} from '@twilio-paste/separator';
+import {Text} from '@twilio-paste/text';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {Box} from '@twilio-paste/box';
+import {Table, THead, TBody, Td, Th, Tr} from '@twilio-paste/table';
+import PackageJSON from '@twilio-paste/core/package.json';
+import {Disclosure, DisclosureContent, DisclosureHeading} from '@twilio-paste/disclosure';
+import {version as coreVersion, dependencies as coreDependencies} from '@twilio-paste/core/package.json';
+import Changelog from '@twilio-paste/core/CHANGELOG.md';
+import {DoDont, Do, Dont} from '../../components/DoDont';
+import {SidebarCategoryRoutes} from '../../constants';
+
+export const pageQuery = graphql`
+  {
+    mdx(fields: {slug: {eq: "/core/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  isCore
+  name={props.pageContext.frontmatter.title}
+  categoryRoute={SidebarCategoryRoutes.CORE}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core"
+  storybookUrl="?path=/story/"
+  packageStatus={[
+    {
+      node: {
+        data: {
+          Documentation: true,
+          Figma: 'not applicable',
+          Design_committee_review: 'yes',
+          Engineer_committee_review: 'yes',
+          Code: 'yes',
+          status: 'production',
+        },
+      },
+    },
+  ]}
+  data={[
+    {
+      node: {
+        name: PackageJSON.name,
+        description: PackageJSON.description,
+        version: PackageJSON.version,
+      },
+    },
+  ]}
+/>
+
+---
+
+ <contentwrapper>
+<content>
+
+## Guidelines
+
+### About Core
+
+<Paragraph>{props.pageContext.frontmatter.description}</Paragraph>
+<Box marginBottom="space100">
+  <Disclosure>
+    <DisclosureHeading as="h4" variant="heading50">
+      Show full package list
+    </DisclosureHeading>
+    <DisclosureContent>
+      <Table tableLayout="fixed">
+        <THead>
+          <Tr>
+            <Th>Package</Th>
+            <Th>Version</Th>
+          </Tr>
+        </THead>
+        <TBody>
+          {Object.keys(coreDependencies).map((dependency, index) => {
+            return (
+              <Tr key={`core-dep-${dependency}-${index}`}>
+                <Td>{dependency}</Td>
+                <Td>{coreDependencies[dependency]}</Td>
+              </Tr>
+            );
+          })}
+        </TBody>
+      </Table>
+    </DisclosureContent>
+  </Disclosure>
+</Box>
+
+We highly recommend accessing Paste components and libraries primarily through Core.
+To do this, install the Core package and use unbarreled imports to individually import
+the components and libraries used in each file.
+
```

Comment:
Hmmm, how about putting [a link to this part of the getting started section](https://paste.twilio.design/getting-started/manual-installation#paste-packages) instead of a code block, to guide people to that `getting started `section if they have reached here before getting there? I'd rather a link bc we do have the 'how to install' section later, in the same place that we have it for other components, and because it does benefit a consumer to read the 'getting started' page before this one. whattaya think?